pytentiostat/main.py

```diff
@@ -0,0 +1,17 @@
+#Pytentiostat function files
+import reporter as reporter
```

Comment:
The code may be easier to read if you use here
```
from reporter  import save_data_to_file
from routines import closing_routines
.
.
.
save_data_to_file(collected_data)
closing_routine(board, d9)
```
I will also look at ``closing_routine`` and see if there is a name that might capture what it does better.

If for some reason you really want to keep the ``reporter`` namespace then you can just say ``import reporter`` rather than ``import reporter as reporter``. It will have the same behavior.   You typically only use ``import...as`` if you want to change the name (usually to shorten it) of the namespace.

Reply:
Yeah I would like this file to be main.py. Also, made the changes to how the functions were imported on the pr and looks cleaner now!